src/screens/SyncOnboarding/index.tsx

```diff
@@ -0,0 +1,226 @@
+                  );
+                  return of(error);
+                }
+                // Disconnection error: retry polling
+                if (
+                  error &&
+                  error instanceof Error &&
+                  error.name === "DisconnectedDevice"
+                ) {
+                  console.log(
+                    `SyncOnboarding: disconnection error 🔌 ${JSON.stringify(
+                      error,
+                    )}`,
+                  );
+                  return of(error);
+                }
+
+                console.log(
+                  `SyncOnboarding: 💥 Error ${error} -> ${JSON.stringify(
+                    error,
+                  )}`,
+                );
+                return throwError(error);
+              }),
+              tap(() => console.log("Going to retry in 🕐️ ...")),
+              delayWhen(() => timer(pollingFrequencyMs)),
+              tap(() => console.log("Retrying 🏃️ !")),
+            ),
+          ),
+          map((deviceVersion: FirmwareInfo) =>
+            extractOnboardingState(deviceVersion.flags),
+          ),
+        )
+        .subscribe({
+          next: (onboardingState: OnboardingState | null) => {
+            console.log(
+              `SyncOnboarding: device version info ${JSON.stringify(
+                onboardingState,
+              )}`,
+            );
+            // FIXME: if null -> initialState ? What should be the initialOnboardingState ?
+            // Does not update the state if it could not be extracted from the flags
+            if (onboardingState) {
+              setOnboardingState(onboardingState);
+            }
+          },
+          error: error =>
+            // TODO: handle error {"name":"DisconnectedDevice","message":"DisconnectedDevice"}
+            console.log(
+              `SyncOnboarding: error ending polling ${error} -> ${JSON.stringify(
+                { error },
+              )}`,
+            ),
+        });
+    }
+
+    return () => {
+      console.log("SyncOnboarding: cleaning up polling 🧹");
+      onboardingStatePollingSubscription?.unsubscribe();
+    };
+  }, [device]);
+
+  // Updates UI step index from the onboarding state
+  useEffect(() => {
+    if (!device) {
+      // No device is paired yet
+      setStepIndex(0);
+    } else if (onboardingState.isConfirmingSeedWords) {
+      setStepIndex(3);
+    }
+    // TODO: cheating - add PIN step once fw has been updated
+    else if (
+      onboardingState.currentSeedWordIndex === 0 &&
+      !onboardingState.isConfirmingSeedWords
+    ) {
+      setStepIndex(1);
+    } else {
+      setStepIndex(2);
+    }
+  }, [onboardingState, device]);
+
+  // When reaching the synchronized onboarding, the device could already
+  // be BLE paired to the phone and at the same time not known by LLM,
+  // it does not matter.
+  // So as long as the device is not onboarded, the LLM will ask for a pairing.
```

Comment:
It will technically not _ask for a pairing_ but rather send the getVersion from https://github.com/LedgerHQ/ledger-live-mobile/pull/2510/files#diff-e4fd4347723086bdb5b2f409bfa15efd6ae4eb4ba7f7fc051c7edb8fe4383ceeR129 and actually work.